### PR TITLE
fix for surrealdb 2.0

### PR DESCRIPTION
**Pull request by RaphaelDarley**
Thank you for submitting this pull request! We appreciate you spending the time to work on these changes.

## Type of Change

- [ ] 📚 Examples / docs / tutorials / dependencies update
- [ ] 🔧 Bug fix (non-breaking change which fixes an issue)
- [ ] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [x] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

## Have you read the [Contributing Guidelines]?

- [x] I have read the [Contributing Guidelines]

[Contributing Guidelines]: https://github.com/surrealdb/surrealdb/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: src/operations/create/core.rs [hunk 1]
```diff
@@ -1,90 +1,152 @@
...
-//! 
+//!
 //! * Create a record in the database
+use crate::connection::interface::WrappedConnection;
 use serde_json::value::Value;
 use surrealdb::opt::Resource;
-use surrealdb::sql::Range;
-use crate::connection::interface::WrappedConnection;
-
+use surrealdb::sql::Thing;
```
Comment (rushmorem) [inline src/operations/create/core.rs:9]: This is no longer a documented API in v2. Consider using `surrealdb::RecordId` instead.

Comment (4F2E4A2E) [reply src/operations/create/core.rs:9]: it should be `surrealdb/surrealdb:v2.0.2`

Comment (RaphaelDarley) [reply src/operations/create/core.rs:9]: This implementation requires accessing fields on Thing, so we have to be exposed to this api

**Comment by unidesigner** [comment]
Hi! I'm using scoped access via the Python SDK on SDB 2.0.4 and I'm looking forward to get this PR merged and released. Thank you!

**Comment by tobiemh** [comment]
Closing this pull-request as the latest codebase now works as a native Python library 🎉

**Comment by 4F2E4A2E** [comment]
When can we count with that release containing the latest codebase?

**Comment by starascendin** [comment]
where is the native Python library?

**Comment by 4F2E4A2E** [comment]
@tobiemh I lii okay rust, but pls have some love for python devs.

**Comment by remade** [comment]
Hey @4F2E4A2E , checked out the latest release?

**Comment by 4F2E4A2E** [comment]
I have subscribed to GitHub release and it says this is the latest release, no?:
https://github.com/surrealdb/surrealdb.py/releases/tag/v0.3.2
image

**Comment by remade** [comment]
@4F2E4A2E See this [tag](https://github.com/surrealdb/surrealdb.py/releases/tag/v0.4.1) and the [Pypi](https://pypi.org/project/surrealdb/0.4.1/) release. The github releases will be looked at

**Pull request closed without merging**